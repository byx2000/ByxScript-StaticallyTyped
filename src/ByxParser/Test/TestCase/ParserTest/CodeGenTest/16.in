int g_i = 100;
double g_x = 3.14;

function int f()
{
	return g_i;
}

function double g()
{
	return g_x;
}

function void main()
{
	
}