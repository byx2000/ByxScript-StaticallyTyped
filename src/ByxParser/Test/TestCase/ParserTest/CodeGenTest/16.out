3
0 6
0 0
0 3

2

igload 0
ret
ret

dgload 1
ret
ret

iconst 100
igstore 0
dconst 3.14
dgstore 1
ret
ret